### PR TITLE
I18n/docs/overview es

### PR DESCRIPTION
**Pull request by AlexScigalszky**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/scullyio/scully/blob/main/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

Introduce a translation of docs/learm/overview.md file 

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Other... Please describe: Translation to spanish

## What is the current behavior?

Translation docs to spanish (one file each time). We will keep the constancy and make it in all files!

Issue Number:  #922

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

## Other information

I translate it be my self but some phases I use google translation (don't worry, it's not a robotic translation)

### CODE REVIEW

File: docs/learn/overview_es.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+title: Learn
+---
+
+# Introducción
+
+## ¿Qué es Scully?
+
+Scully es el mejor generador de web estática para proyectos Angular y así adoptar una arquitectura [Jamstack](https://jamstack.org/).
+
+A partir de tu aplicación creará un archivo estático `index.html` por cada pages/routes. Todos y cada uno de los `index.html` tendrán el contenido ya listo, y esto hará que tu aplicación se muestre instantáneamente a los usuarios. También, permitirá que tu aplicación mejore el SEO. Por ensima de todo esta funcionalidad, tu SPA mantendrá todas las funcionalidades que tenía anteriormente.
```
Comment (jorgeucano) [inline docs/learn/overview_es.md:14]: encima

File: docs/learn/overview_es.md [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+Para agregarlo a tu proyecto y ejecutarlo por primera vez:
+
+```bash
+ng add @scullyio/init
+ng build --prod
+npm run scully
+```
+
+Para una explicación más detallada vea nuestro getting started.
```
Comment (jorgeucano) [inline docs/learn/overview_es.md:37]: No estoy seguro de usar "getting started" deberiamos traducirlo tambien

**Review by jorgeucano — COMMENTED** [review]
Hey @AlexScigalszky thanks for the PR , I just add 2 comments, can you fix that? And I will merge :D

File: docs/learn/overview_es.md (hunk 1 above)
Comment (AlexScigalszky) [reply docs/learn/overview_es.md]: done

File: docs/learn/overview_es.md (hunk 2 above)
Comment (AlexScigalszky) [reply docs/learn/overview_es.md]: done

**Review by jorgeucano — APPROVED** [review]

**Pull request merged**